cmd/juju/status/status_test.go

```diff
@@ -171,8 +171,9 @@ var (
 			"current": "started",
 			"since":   "01 Apr 15 01:23+10:00",
 		},
-		"dns-name":    "controller-0.dns",
-		"instance-id": "controller-0",
+		"dns-name":     "controller-0.dns",
+		"ip-addresses": []string{"controller-0.dns"},
```

Comment:
I'm confused by the names we use. If it is `address` then why aren't they IP addresses? Similarly, if we don't want addresses then let's use `hostnames`.